index.html

```diff
@@ -124,6 +137,20 @@ <h1 class="details-title">Exercise Name</h1>
           <button class="blue-btn">Add to Workout</button>
         </div>
       </section>
+      <section
+        class="favorites-view flex flex-col align-center hidden"
+        data-view="favorites-view">
+        <h1>Favorite Exercises</h1>
+        <div
+          class="card-list flex wrap align-center"
+          id="favorites-card-list"></div>
+        <div class="no-results flex flex-col align-center hidden">
+          <p class="" style="">No Results Found.</p>
+          <img
```

Comment:
Instead of saying "No results found" here, I would put a call to action. Something like "No favorites exercises yet." and then a link back to your search page that looks like a button that says "Search for exercises"